src/main/webapp/WEB-INF/i18n/messages_ar.properties

```diff
@@ -24,6 +24,7 @@ application.content=محتوى التطبيق
 application.opened.events=Application opened events
 application.versions=Application versions
 applications=Applications
+app.categories=App categories
```

Comment:
Doesn't look Arabic to me :)

Reply:
@sladomic Will have a native speaker go through and translate these later ;-)